app/src/main/java/ch/epfl/skysync/screens/LoginScreen.kt

```diff
@@ -44,7 +53,10 @@ fun LoginScreen(signInLauncher: ActivityResultLauncher<Intent>, user: FirebaseUs
 
           if (user != null) {
             // logged in
-            Text(text = "Welcome")
+            navController.navigate(Route.HOME) {
```

Comment:
Is there a reason why the `navigate` function is called here instead of in the `MainActivity.onSignInResult` callback ?

Reply:
My problem is how to deal with someone who is already connected. With the actual implementation it does not seem possible but it would be suitable that we do not have to connect every time we access the app. So I haved two choices either making it this way or put a condition on the start destination of the navigation graph. What do you think is better ?